addon/options/options.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+  </head>
+
+  <body>
+    <form>
+      <label for="environment">Environment:</label>
+      <select name="environment" id="environment">
+        <option value="https://example.com:8000" selected>Local</option> 
+        <option value="https://testpilot.dev.mozaws.net">Development</option>
+        <option value="https://testpilot-l10n.dev.mozaws.net">L10n</option>
+        <option value="https://testpilot.stage.mozaws.net">Stage</option>
+        <option value="https://testpilot.firefox.com">Production</option>

```

Comment:
since production is the default in background.js I think it should be the one initialized to selected here